docs/android/submit.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Android Submit Contribution"
+sidebar_label: "Submit Contribution"
+---
+
+## Submit Your First Contribution
+
+Congratulations on completing your first version! Now it's time to get feedback and prepare your work for real users. Follow the [GitHub Documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) to create a pull request (PR) from your fork.
+
+### Checklist for Submitting a PR
+
+When creating a PR, GitHub pre-fills the description with a checklist. Ensure you follow all the steps. Here's an extended checklist to help you:
+
+- ✅ **PR Description**: Provide a clear and complete description of your changes.
+- ✅ **Tests**: Add all necessary tests following our [testing guidelines](testing/introduction).
+- ✅ **Documentation**: Ensure your code is properly documented.
+- ✅ **UI Changes**: Include screenshots if the UI is modified.
+- ✅ **User Documentation**: If user documentation needs updates, open a PR on [GitHub](https://github.com/home-assistant/companion.home-assistant).
```

Comment:
Should we also mention the beta label here as that is usually missed in most PRs adding features :)

Reply:
Not sure how to formulate it. any suggestion?

Reply:
maybe a sub-bullet there with a quick mention of adding the beta label for any PR that adds or updates a feature? Thats generally when we need to add beta labels. The purpose is really for beta users to be able to find docs on unreleased features.

Reply:
I wonder if this should not be in the PR checklist for Android and iOS, maybe even core?

Reply:
I made a proposal in https://github.com/home-assistant/android/pull/5199